File: TDS/TDS/Models/Entrega.cs
using System;
using System.Collections.Generic;

namespace TDS.Models
{
    public partial class Entrega
    {
        public int Id { get; set; }
        public string? Documento { get; set; }
        public int? TareaId { get; set; }
        public int? EstudianteId { get; set; }
        public bool? Estado { get; set; }

        public virtual Estudiante? Estudiante { get; set; }
        public virtual Tarea? Tarea { get; set; }
    }
}

File: TDS_Front/src/app/models/entrega.ts
import { Estudiante } from "./estudiante";
import { Tarea } from "./tarea";

export class Entrega {
  constructor(public id: number = 0, public documento: string = "", public tareaId: number = 0, public estudianteId: number | null = 0, public estado: boolean = false, public calificacion: number = 0, public estudiante: Estudiante | null = new Estudiante(), public tarea: Tarea | null = new Tarea()){}
}
